content/post/pga_history.md

```diff
@@ -0,0 +1,91 @@
+* Some repositories have packfiles in the [old deprecated format]. go-git handles them.(https://github.com/git/git/blob/master/Documentation/technical/pack-format.txt#L39).
+* Load balancing is difficult. 90% of the repositories are compact and fly fast, however, the rest 10% occupy 50% of the dataset size and are damn slow to process.
+* Analyzing big repositories requires much memory, as much as 128 GB. Thus our pipeline kept exploding again and again and we restarted it with a firm hand.
+
+We had to take into account the removed repositories. Over 3,000 were deleted within few months.
+We discovered a few repositories with porno while inspecting the outliers by repository size (no links here, sorry, join our community Slack and direct message).
+
+Overall, our Data Retrieval team did a fantastic job. It was great to see how
+the numerous open source libraries they developed played together to deliver the dataset.
+
+## Paper
+
+As for the mining of Git repositories, we attended
+[Mining Software Repositories](http://2017.msrconf.org) (MSR) conference in 2017
+and loved it. There was an opportunity to submit a paper about our dataset
+to the "Data Showcase" track on [MSR'18](http://2018.msrconf.org).
+We took action, wrote the paper and luckily for us it was accepted.
+
+<embed src="https://arxiv.org/pdf/1803.10144" width="600" height="600" alt="PGA paper from ArXiV">
+
+{{% caption src="/post/pga_history/presentation.jpg" %}}
+Vadim is presenting Public Git Archive on MSR'18/ICSE'18.
+{{% /caption %}}
+
+We used [Overleaf](https://www.overleaf.com/) to write the paper. When we reached the internal review
+phase, we pushed it to a private GitHub repository and the rest of the edits happenned in pull requests.
+Thus we avoided the "cold start" problem where the first pull request is huge and nightmare to read and collaborate. If we did not have to use TeX, we would use Google Docs instead.
+
+It was hard to squeeze the content into 4 pages. The layout constantly broke, we had to "fix" it with
+`\vspace`. Figures tended to appear on a different page from the related text so we forced the placement
+and got the "jumping" problem instead. The bibliography was very verbose and we had to "optimize" the records
+to make them shorter and consequently fit into one column.
+
+## Poster
+
+We presented the poster about PGA on [MSR'18](http://2018.msrconf.org) poster session.
+It was created in Adobe Illustrator and Inkscape.
+
+<embed src="/post/pga_history/pga_poster.pdf" width="600" height="600" alt="PGA poster for MSR'18">
+
+{{% caption src="/post/pga_history/poster.jpg" %}}
+Conference attendees looking at the PGA poster on MSR'18/ICSE'18.
+{{% /caption %}}
+
+## Future
+
+We plan to release the next version of PGA later this year or early next year.
+It is included into our [Q4 OKRs](https://github.com/src-d/okrs/tree/2018Q4#release-a-new-version-of-pga-infraapps).
+Stay tuned.
```

Comment:
I would finish with a call to action. Maybe something regarding on how to learn more about PGA and start using it.